src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
thanks for your examples... now I get whats going on and I agree we need to refactor things, because it is misleading, at least for non-js devs ;-).

But hey, we had unit tests which prevented that I am doing stupid things... ;-)